gallery/src/pages/AdditionalResources.tsx

```diff
@@ -0,0 +1,109 @@
+          Official ReactJS Tutorial
+        </a>
+        {' '}(Beginner Level)
+      </li>
+      <li className="nx-list__item">
+        <a rel="noreferrer"
+           href="https://redux.js.org/basics/basic-tutorial"
+           target="_blank">Official Redux Getting Started
+        </a>
+        {' '}(Beginner Level)
+      </li>
+      <li className="nx-list__item">
+        <a rel="noreferrer"
+           href="https://egghead.io/courses/getting-started-with-redux"
+           target="_blank">
+          Getting started with Redux
+        </a>
+        {' '}(egghead.io beginner course)
+      </li>
+      <li className="nx-list__item">
+        <a rel="noreferrer"
+           href="https://www.pluralsight.com/paths/react"
+           target="_blank">
+             Pluralsight React Path
+        </a>
+        {' '}(beginner to advanced)
+      </li>
+      <li className="nx-list__item">
+        <a rel="noreferrer"
+           href="https://reactforbeginners.com/"
+           target="_blank">
+             React for beginners
+        </a>
+      </li>
+    </ul>
+
+    <h3 className="nx-h3 nx-tile__section-header">Good Reads</h3>
+    <ul className="nx-list nx-list--bulleted">
+      <li className="nx-list__item">
+        <a rel="noreferrer"
+           href="https://reactjs.org/docs/hello-world.html"
+           target="_blank">Main Concepts in React
+        </a>
+      </li>
+      <li className="nx-list__item">
+        <a rel="noreferrer" href="https://reactjs.org/docs/thinking-in-react.html" target="_blank">Thinking in React</a>
+      </li>
+      <li className="nx-list__item">
+        Many apps utilizing React and Redux together utilize the official{' '}
+        <a rel="noreferrer" href="https://react-redux.js.org/using-react-redux/connect-mapstate" target="_blank">
+          react-redux
+        </a>
+        {' '}connectors, as well as the{' '}
+        <a rel="noreferrer" href="https://redux-toolkit.js.org/introduction/quick-start" target="_blank">
+          Redux toolkit
+        </a>
+        , which is a set of patterns to reduce boilerplate in “vanilla Redux”.
+      </li>
+      <li className="nx-list__item">
+        <a rel="noreferrer"
+           href="https://www.youtube.com/watch?v=nYkdrAPrdcw&time_continue=781"
+           target="_blank">
+          Video about Flux
+        </a>
+        , which is a precursor to redux. Uses some real-world problems to motivate the concept of one-way dataflow for
+        state management, which is what Redux is all about.
+      </li>
+    </ul>
+
+    <h3 className="nx-h3 nx-tile__section-header">Getting your hands dirty</h3>
+    <ul className="nx-list nx-list--bulleted">
+      <li className="nx-list__item">
+        <a rel="noreferrer"
+           href="https://github.com/sonatype/sonatype-application-builder"
+           target="_blank">
+             Sonatype Application Builder (SAB) Template
```

Comment:
Maybe add a note that this is an internal Sonatype only project?

Reply:
done in 18a0d7d